Documentation/api-reference.md

```diff
@@ -1,4 +1,4 @@
-# Adobe Experience Platform Media for Edge Network Extension - iOS
+# Adobe Streaming Media for Edge Network extension - iOS
 
 ## Prerequisites
 
```

Comment:
Update L12 and L67 
registerExtension**s**

Reply:
Add toc and prerequisites - e.g. https://github.com/adobe/aepsdk-edgeidentity-android/blob/main/Documentation/api-reference.md